docs/tutorial/part-four/index.md

```diff
@@ -274,7 +272,33 @@ export const query = graphql`
 `
 ```
 
-`src/components/layout.js`
+It worked!! 🎉
+
+![fake-title-graphql](fake-title-graphql.png)
+
+Page queries must live outside of the source code, by convention at the end of a
```

Comment:
"outside of the source code" => "outside of the component definition"

"by convention at the end of a page" => "by convention at the end of the page component file"

just to clarify a bit more

Reply:
👍